test/integration/binary/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+
+if (SWIG_PYTHON OR SWIG_JAVA)
```

Comment:
Consider adding a variable `IS_BINDINGS` or `SWIG_ENABLED`, etc
otherwise it be duplicated in bindings folder as well

Reply:
at least mark with a todo, same for the following one

Reply:
Discussed offline. Not a problem.